[2022-05-23 12:25:55] [WARNING]: 警告: 配置文件内的QQ号 (1233456) 与缓存内的QQ号 (2945339871) 不相同 
[2022-05-23 12:25:55] [WARNING]: 1. 使用会话缓存继续. 
[2022-05-23 12:25:55] [WARNING]: 2. 删除会话缓存并重启. 
[2022-05-23 12:25:55] [WARNING]: 请选择: (5秒后自动选1) 
[2022-05-23 12:26:59] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 12:27:03] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 17:57:51] [WARNING]: 警告: 配置文件内的QQ号 (1233456) 与缓存内的QQ号 (2945339871) 不相同 
[2022-05-23 17:57:51] [WARNING]: 1. 使用会话缓存继续. 
[2022-05-23 17:57:51] [WARNING]: 2. 删除会话缓存并重启. 
[2022-05-23 17:57:51] [WARNING]: 请选择: (5秒后自动选1) 
[2022-05-23 18:00:21] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:00:25] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:00:29] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:09:11] [WARNING]: 警告: 配置文件内的QQ号 (1233456) 与缓存内的QQ号 (2945339871) 不相同 
[2022-05-23 18:09:11] [WARNING]: 1. 使用会话缓存继续. 
[2022-05-23 18:09:11] [WARNING]: 2. 删除会话缓存并重启. 
[2022-05-23 18:09:11] [WARNING]: 请选择: (5秒后自动选1) 
[2022-05-23 18:09:26] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:12:51] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:13:16] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:13:20] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:13:24] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:13:28] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":42,"PacketSent":28,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300794}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:13:30] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:13:34] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:13:38] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:13:42] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":43,"PacketSent":29,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300808}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:13:44] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:13:48] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:13:52] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:13:55] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":43,"PacketSent":29,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300822}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:13:58] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:14:02] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:14:05] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:14:09] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":44,"PacketSent":30,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300835}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:14:11] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:14:15] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:14:19] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:14:23] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":44,"PacketSent":30,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300849}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:14:25] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:14:29] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:14:33] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:14:37] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":45,"PacketSent":31,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300863}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:14:39] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:14:43] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:14:47] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:14:50] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":45,"PacketSent":31,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300877}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:14:53] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:14:56] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:15:00] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:15:04] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":46,"PacketSent":32,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300890}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-05-23 18:15:06] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-05-23 18:15:10] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-05-23 18:15:14] [WARNING]: 上报 Event 数据到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-05-23 18:15:18] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":2945339871,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":46,"PacketSent":32,"PacketLost":0,"MessageReceived":3,"MessageSent":3,"LastMessageTime":1653300786,"DisconnectTimes":0,"LostTimes":0}},"time":1653300904}
 到 http://localhost:8400/ 失败: Post "http://localhost:8400/": dial tcp [::1]:8400: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
